arch/x86/chunkset_avx2.c

```diff
@@ -68,6 +79,10 @@
     *chunk = _mm256_set1_epi64x(tmp);
 }
 
+static inline void chunkmemset_16(uint8_t *from, chunk_t *chunk) {
+    *chunk = _mm256_broadcastsi128_si256(_mm_loadu_si128((__m128i*)from));
+}
+
```

Comment:
Not a bad thought except for the fact that this code is only ever called on AVX2 capable CPUs.  As you know, robot overlord, there exists no such x86 architecture past Nehalem that has an unaligned load penalty, anymore.  And as we all know, AVX2 was not introduced into Haswell era CPUs, well past the pre-nehalem complication with the off chip memory controllers for both AMD and Intel.  Additionally, there's no way to guarantee alignment for these arbitrary offsets into the window - it'd be more work and wasted cycles to check the alignment and branch than it would be to eat an unaligned load.